### ISSUE TITLE
BUILD FAILED Target "aar" does not exist

### ISSUE THREAD

**Issue by fink013** [LWJGL/android-test#8]
> BUILD FAILED
> Target "aar" does not exist in the project "LWJGL".

ran in "android-test/lwjgl"

Any ideas how to create that aar?

**Comment by JoshBot-Debug** [comment]
Did you find out what the problem was? I'm having the same issue here.

**Comment by fink013** [comment]
Unfortunately not. I switched to LibGDX which also includes some LWJGL features.

**Comment by JoshBot-Debug** [comment]
Yeah, I just dropped the LWJGL idea and moved to LibGDX, are you using a linux?

**Comment by fink013** [comment]
Yes, Ubuntu with IntelliJ.

**Comment by typhonrt** [comment]
I can't directly help, but have some long term interest in seeing Android / particularly Vulkan on Android support continue. It's been a while since this repo has been updated, so some of the tooling / gradle, etc. or something else of that nature might need to be updated. I know @Spasi does great work and the main LWJGL repos are still active, so perhaps he'll make it around to this area again somewhat soon.
